tests/migrations/helpers/test_update_legacy_filtered_positives_helper.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import numpy as np
+from migrations.helpers.update_legacy_filtered_positives_helper import (
+    v0_version_set,
+    legacy_mongo_samples,
+    get_cherrypicked_samples_by_date,
+    split_mongo_samples_by_version,
+)
+from crawler.constants import (
+    FIELD_PLATE_BARCODE,
+    FIELD_ROOT_SAMPLE_ID,
+    V0_V1_CUTOFF_TIMESTAMP,
+    V1_V2_CUTOFF_TIMESTAMP,
+)
+
+# ----- migration helper function tests -----
+
+
+def test_legacy_mongo_samples_returns_correct_samples_filtered_by_date(
+    config, filtered_positive_testing_samples
+):  # noqa: E501
+    result = legacy_mongo_samples(config)
+    assert len(result) == 3
+
+
+def test_check_versions_set_returns_true_with_v0(config, filtered_positive_testing_samples):
+    assert v0_version_set(config) is True
+
+
+def test_check_versions_set_returns_false_with_no_v0_samples(
+    config, filtered_positive_testing_samples_no_v0
+):  # noqa: E501
+    assert v0_version_set(config) is False
+
+
+def test_get_cherrypicked_samples_by_date_v0_returns_expected(
+    config, event_wh_data, mlwh_sample_stock_resource, mlwh_sql_engine, event_wh_sql_engine
```

Comment:
You're right about them not being needed here, I've removed them.

For the way that `event_wh_data` is set up in conftest.py, it assumes that the engine is a fixture (this is the same in Lighthouse). I **think** that you could change the way it's used in there so that it's just a function and not a fixture. Might then be useful to update in Lighthouse as well, so that they are consistent. Worth doing?

Reply:
Nah keep it as a test fixture. I don't think there is anything wrong with test fixture inheritance (of sorts)